inc/core/patterns/bcgov-story-with-dark-bg.php

```diff
@@ -0,0 +1,88 @@
+	<div class="wp-block-media-text alignwide has-media-on-the-right is-stacked-on-mobile is-img-rounded is-style-square has-accent-stroke has-white-color has-primary-brand-background-color has-text-color has-background has-link-color" style="margin-top:6rem;margin-bottom:6rem;grid-template-columns:auto 40%"><div class="wp-block-media-text__content"><!-- wp:group {"style":{"spacing":{"padding":{"right":"2rem"}}},"layout":{"type":"constrained"}} -->
+	<div class="wp-block-group" style="padding-right:2rem"><!-- wp:image {"id":3607,"width":"120px","sizeSlug":"full","linkDestination":"none","className":"icon"} -->
+	<figure class="wp-block-image size-full is-resized icon"><img src="' . esc_url( get_template_directory_uri() ) . '/assets/images/block-theme/icon_social.png" alt="" class="wp-image-3607" style="width:120px" title=""/></figure>
+	<!-- /wp:image -->
+	
+	<!-- wp:heading {"level":3,"style":{"elements":{"link":{"color":{"text":"var:preset|color|white"}}},"spacing":{"margin":{"top":"0","bottom":"0"}}},"textColor":"white","fontSize":"extra-large"} -->
+	<h3 class="wp-block-heading has-white-color has-text-color has-link-color has-extra-large-font-size" style="margin-top:0;margin-bottom:0">Story Section 2 Heading</h3>
+	<!-- /wp:heading -->
+	
+	<!-- wp:paragraph {"style":{"typography":{"fontSize":"1.15rem"}}} -->
+	<p style="font-size:1.15rem">Cras justo odio, dapibus ac facilisis in, egestas eget quam. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec ullamcorper nulla non metus auctor fringilla. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Maecenas sed diam eget risus varius blandit sit amet non magna. Donec id elit non mi porta gravida at eget metus. Cras justo odio, dapibus ac facilisis in, egestas eget quam.</p>
+	<!-- /wp:paragraph -->
+	
+	<!-- wp:heading {"level":4,"style":{"elements":{"link":{"color":{"text":"var:preset|color|white"}}},"spacing":{"margin":{"bottom":"0"}},"typography":{"fontStyle":"normal","fontWeight":"400"}},"textColor":"white","fontSize":"large"} -->
+	<h4 class="wp-block-heading has-white-color has-text-color has-link-color has-large-font-size" style="margin-bottom:0;font-style:normal;font-weight:400">Story Section 2 Additional Info</h4>
+	<!-- /wp:heading -->
+	
+	<!-- wp:paragraph {"style":{"typography":{"fontSize":"1.15rem"}}} -->
+	<p style="font-size:1.15rem">Positive brand and organizational reputation, risk reduction, attracting capital, and fostering positive relationships with stakeholders all contribute to long-term sustainability for a company. A commitment to social responsibility better positions a business to thrive in a changing environment and to meet increasing demands for ethical and sustainable practices in business.</p>
```

Comment:
Text seems at odds with the use of Lorem in other places.

Reply:
Fixed